src/main/scala/io/iohk/ethereum/jsonrpc/TestService.scala

```diff
@@ -131,6 +130,8 @@ class TestService(
   private var currentConfig: BlockchainConfig = initialConfig
   private var blockTimestamp: Long = 0
   private var sealEngine: SealEngineType = SealEngineType.NoReward
+  private val preimageCache: collection.concurrent.Map[ByteString, UInt256] =
```

Comment:
Looks like this `preimageCache` is used to store only storage data. If that's the case it would make sense to make this obvious from the type definition by using an `AnyVal` wrappers for `ByteString` and `UInt256` like say `StorageHash`, `StorageKey`